test/unit_tests/test_client.py

```diff
@@ -0,0 +1,36 @@
+"""Unit-тесты клиента"""
+
+import sys
+import os
+import unittest
+sys.path.append(os.path.join(os.getcwd(), '..'))
+from common.variables import RESPONSE, ERROR, USER, ACCOUNT_NAME, TIME, ACTION, PRESENCE
+from client import create_presence, process_ans
+
+class TestClass(unittest.TestCase):
+    '''
+    Класс с тестами
+    '''
+
+    def test_def_presense(self):
+        """Тест коректного запроса"""
+        test = create_presence()
+        test[TIME] = 1.1  # время необходимо приравнять принудительно
+                          # иначе тест никогда не будет пройден
+        self.assertEqual(test, {ACTION: PRESENCE, TIME: 1.1, USER: {ACCOUNT_NAME: 'Guest'}})
+
+    def test_200_ans(self):
+        """Тест корректтного разбора ответа 200"""
+        self.assertEqual(process_ans({RESPONSE: 200}), '200 : OK')
+
+    def test_400_ans(self):
+        """Тест корректного разбора 400"""
+        self.assertEqual(process_ans({RESPONSE: 400, ERROR: 'Bad Request'}), '400 : Bad Request')
+
+    def test_no_response(self):
+        """Тест исключения без поля RESPONSE"""
+        self.assertRaises(ValueError, process_ans, {ERROR: 'Bad Request'})

```

Comment:
```
  File "C:\Projects\Python\Client-server-applications-in-python\test\unit_tests\..\client.py", line 69, in process_response_ans
    raise ReqFieldMissingError(RESPONSE)
errors.ReqFieldMissingError: В принятом словаре отсутствует обязательное поле response.
```

Reply:
The tests don't need to be changed because on the lesson it was said hat they won't be needed anymore